{
  "modified": "2024-10-16T14:45:53Z",
  "published": "2024-10-16T14:45:53Z",
  "schema_version": "1.5.0",
  "id": "",
  "aliases": [
    "MAL-2024-5520"
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pipfont"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/pipfont/MAL-2024-5520.json"
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "26c264b90c20b833af34baa73d3fa2f9d2d49db704a89c4a482a429570818ff2",
        "import_time": "2024-10-24T00:59:38.065171545Z",
        "id": "RLUA-2024-08730",
        "modified_time": "2024-10-16T14:45:53Z"
      }
    ]
  }
}
